week-1/exercises/decimal.js

```diff
@@ -1,5 +1,13 @@
 const num = 58.4567;
 
+const wholeNumberPart = Math.floor(num) - 2; // I had to subtract 2 to get wholeNumberPart = 56
+
+const decimalPart = parseFloat((num % 1).toFixed(4)) // to fix Decimal Part: 0.4566999999999979 that only evaluates to 0.4567  ;
```

Comment:
this works, but it's a little trickly to read. 

`const decimalPart = num - wholeNumberPart` is a lot simpler. 

Remember - code is read far more often than it is written